src/app/vendor/videojshooks.js

```diff
@@ -86,24 +86,27 @@ vjs.TextTrack.prototype.load = function () {
         this.readyState_ = 1;
 
         var subsParams = function () {
-            $('#video_player .vjs-text-track').css('display', 'inline-block').drags();
-            $('#video_player .vjs-text-track-display').css('font-size', Settings.subtitle_size);
+            var subtitles = $('.vjs-subtitles');
+            var vjsTextTrack = $('.vjs-text-track');
+
+            vjsTextTrack.css('display', 'inline-block').drags();
+            vjsTextTrack.css('font-size', Settings.subtitle_size);
             if (win.isFullscreen) {
-                $('.vjs-text-track').css('font-size', '140%');
+                vjsTextTrack.css('font-size', '140%');
```

Comment:
What this is doing ? is it overwriting the defined subtitles font size in settings ?

Reply:
This functionality was already here and it looks like it sets bigger font size for full screen mode. As of my changes I fixed duplicate jQuery selector since jQuery doesn't cache selectors.

Reply:
Ok, thx